app/serializers/activity_serializer.rb

```diff
@@ -17,8 +17,8 @@
 #
 class ActivitySerializer
   include JSONAPI::Serializer
-  attributes :name, :content
 
+  attributes :name, :content
   attributes :image_url do |activity|
```

Comment:
I think that is better to name this as `:image` and change to has_one_attached image to all models to avoid any confusion.